src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java

```diff
@@ -80,7 +80,7 @@ public void upload(LocalSourceFile file, boolean isSmokeTestDoc) {
 
                 return sftp.ls(configProperties.getReportsFolder())
                     .stream()
-                    .filter(RemoteResourceInfo::isRegularFile)
+                    .filter(f -> f.isRegularFile() && f.getName().toLowerCase().endsWith(".csv"))
```

Comment:
not sure whether `RemoteResourceInfo` became unused import but ci will pick this up - not a blocker

Reply:
Well spotted. Removed the import.

Reply:
how about 2 filters?